### ISSUE TITLE
invalid HTTPS certificate when visiting directly https://www.retroshare.cc

### ISSUE THREAD

**Issue by pedro-nonfree** [RetroShare/RetroShare#1737]
Visiting https://www.retroshare.cc

Chromium says

> This server could not prove that it is www.retroshare.cc; its security certificate is from www.github.com. This may be caused by a misconfiguration or an attacker intercepting your connection.

Firefox says

> Websites prove their identity via certificates. Firefox does not trust this site because it uses a certificate that is not valid for www.retroshare.cc. The certificate is only valid for the following names: www.github.com, *.github.io, *.githubusercontent.com, *.github.com, github.com, github.io, githubusercontent.com

**Comment by csoler** [comment]
Try https://retroshare.cc

**Comment by pedro-nonfree** [comment]
> Try https://retroshare.cc

yes, I know that domain URL works. Just saying that [this domain](https://www.retroshare.cc) is invalid and should be fixed. The most easy thing is a redirection from https://www.retroshare.cc to https://retroshare.cc

**Comment by felisucoibi** [comment]
agree with pedro must be fixed

**Comment by defnax** [comment]
@csoler @thunder2 close
